### PR TITLE
Add some older changelogs in pretty markdown

### PR DESCRIPTION
**Pull request by MarcelMue**
Just added more changelogs from older versions to make it easier to find them.

### CODE REVIEW

File: release-notes/aws/v9.0.0.md [hunk 1]
```diff
@@ -23,7 +23,7 @@
   - The default value of configmap setting `proxy-add-original-uri-header` is now `false`.
   - When the setting `proxy-add-original-uri-header` is `true`, the ingress controller adds a new header `X-Original-Uri` with the value of NGINX variable `$request_uri`. In most of the cases this is not an issue but with request with long URLs it could lead to unexpected errors in the application defined in the `Ingress` `serviceName`, like issue https://github.com/kubernetes/ingress-nginx/issues/4593.
 
-### coredns v1.6.4 ([GS v0.8.0](https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v080))
+### coreDNS v1.6.4 ([GS v0.8.0](https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v080))
```
Comment (kopiczko) [inline release-notes/aws/v9.0.0.md]: How about putting those links on the bottom of the file like here https://raw.githubusercontent.com/giantswarm/architect-orb/master/CHANGELOG.md ?

File: release-notes/azure/v8.4.0.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+## :zap:  Giant Swarm Release 8.4.0 for Azure is now active for you! :zap:
+
+This release contains Kubernetes 1.14.6 and therefore fixes for CVE-2019-9512 and CVE-2019-9514.
+
+### Kubernetes v1.14.6
+- Updated from v1.14.5 - [changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md#v1146)
```
Comment (kopiczko) [inline release-notes/azure/v8.4.0.md]:
```suggestion
- Update from v1.14.5 - [changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md#v1146).
```

File: release-notes/azure/v8.4.0.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- Includes CVE-2019-9512, CVE-2019-9514
```
Comment (kopiczko) [inline release-notes/azure/v8.4.0.md]:
```suggestion
- Include CVE-2019-9512, CVE-2019-9514.
```

File: release-notes/azure/v8.4.0.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+### kube-state-metrics v1.7.2 ([GS v0.4.0](https://github.com/giantswarm/kube-state-metrics-app/blob/master/CHANGELOG.md#v040))
+- Updated from v1.6.0 - [changelog](https://github.com/kubernetes/kube-state-metrics/releases/tag/v1.7.2)
```
Comment (kopiczko) [inline release-notes/azure/v8.4.0.md]:
```suggestion
- Update from v1.6.0 - [changelog](https://github.com/kubernetes/kube-state-metrics/releases/tag/v1.7.2).
```

File: release-notes/azure/v9.0.0.md [hunk 5]
```diff
@@ -29,5 +29,5 @@ This release contains Kubernetes `1.15.5` as well as a change to the default wor
   - The default value of configmap setting `proxy-add-original-uri-header` is now `false`.
   - When the setting `proxy-add-original-uri-header` is `true`, the ingress controller adds a new header `X-Original-Uri` with the value of NGINX variable `$request_uri`. In most of the cases this is not an issue but with request with long URLs it could lead to unexpected errors in the application defined in the `Ingress` `serviceName`, like issue https://github.com/kubernetes/ingress-nginx/issues/4593.
 
-### coredns v1.6.4 ([GS v0.8.0](https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v080))
-- Updated from upstream `coredns` v1.6.2 - [changelog](https://coredns.io/2019/09/27/coredns-1.6.4-release/)
+### coreDNS v1.6.4 ([GS v0.8.0](https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v080))
+- Updated from upstream `coreDNS` v1.6.2 - [changelog](https://coredns.io/2019/09/27/coredns-1.6.4-release/)
```
Comment (kopiczko) [inline release-notes/azure/v9.0.0.md]:
```suggestion
- Update from upstream `coreDNS` v1.6.2 - [changelog](https://coredns.io/2019/09/27/coredns-1.6.4-release/).
```

File: release-notes/kvm/v8.4.0.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
+## :zap:  Giant Swarm Release 8.4.0 for KVM is now active for you! :zap:
+
+This release contains Kubernetes 1.14.6 and therefor fixes for CVE-2019-9512 and CVE-2019-9514.
+
+### Kubernetes v1.14.6
+- Update kubernetes to 1.14.6 (CVE-2019-9512, CVE-2019-9514) - [changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md#v1146)
```
Comment (kopiczko) [inline release-notes/kvm/v8.4.0.md]:
```suggestion
- Update kubernetes to 1.14.6 (CVE-2019-9512, CVE-2019-9514) - [changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md#v1146).
```

File: release-notes/kvm/v8.4.0.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+### kvm-operator v3.8.0
+- Fix NTP server configuration on tenant nodes.
+
+### coreDNS v1.6.2 ([GS 0.7.0](https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v070))
+- Updated from 1.5.1 - [changelog](https://coredns.io/2019/08/13/coredns-1.6.2-release/)
```
Comment (kopiczko) [inline release-notes/kvm/v8.4.0.md]:
```suggestion
- Updated from 1.5.1 - [changelog](https://coredns.io/2019/08/13/coredns-1.6.2-release/).
```

File: release-notes/kvm/v8.4.0.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+### kube-state-metrics v1.7.2 ([GS 0.4.0](https://github.com/giantswarm/kubernetes-kube-state-metrics/blob/master/CHANGELOG.md#v040))
+- Updated from 1.6.0 - [changelog](https://github.com/kubernetes/kube-state-metrics/releases/tag/v1.7.2)
+
+### ignition  
+- Add name label for default and kube-system namespaces.
+- Use v1 stable for giantswarm-critical priority class.
+- Introduce explicit resource reservation for OS resources and container runtime.
+
+### Calico v3.8.2
+- Update calico to 3.8.2 - [changelog](https://docs.projectcalico.org/v3.8/release-notes/)
```
Comment (kopiczko) [inline release-notes/kvm/v8.4.0.md]:
```suggestion
- Update calico to 3.8.2 - [changelog](https://docs.projectcalico.org/v3.8/release-notes/).
```

File: release-notes/kvm/v8.4.0.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+- Update kubernetes to 1.14.6 (CVE-2019-9512, CVE-2019-9514) - [changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md#v1146)
+
+### kvm-operator v3.8.0
+- Fix NTP server configuration on tenant nodes.
+
+### coreDNS v1.6.2 ([GS 0.7.0](https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v070))
+- Updated from 1.5.1 - [changelog](https://coredns.io/2019/08/13/coredns-1.6.2-release/)
+
+### kube-state-metrics v1.7.2 ([GS 0.4.0](https://github.com/giantswarm/kubernetes-kube-state-metrics/blob/master/CHANGELOG.md#v040))
+- Updated from 1.6.0 - [changelog](https://github.com/kubernetes/kube-state-metrics/releases/tag/v1.7.2)
```
Comment (kopiczko) [inline release-notes/kvm/v8.4.0.md]:
```suggestion
- Update from 1.6.0 - [changelog](https://github.com/kubernetes/kube-state-metrics/releases/tag/v1.7.2).
```

File: release-notes/kvm/v9.0.0.md [hunk 10]
```diff
@@ -23,5 +23,5 @@
   - The default value of configmap setting `proxy-add-original-uri-header` is now `false`.
   - When the setting `proxy-add-original-uri-header` is `true`, the ingress controller adds a new header `X-Original-Uri` with the value of NGINX variable `$request_uri`. In most of the cases this is not an issue but with request with long URLs it could lead to unexpected errors in the application defined in the `Ingress` `serviceName`, like issue https://github.com/kubernetes/ingress-nginx/issues/4593.
 
-### coredns v1.6.4 ([GS v0.8.0](https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v080))
-- Updated from upstream `coredns` v1.6.2 - [changelog](https://coredns.io/2019/09/27/coredns-1.6.4-release/)
+### coreDNS v1.6.4 ([GS v0.8.0](https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v080))
+- Updated from upstream `coreDNS` v1.6.2 - [changelog](https://coredns.io/2019/09/27/coredns-1.6.4-release/)
```
Comment (kopiczko) [inline release-notes/kvm/v9.0.0.md]:
```suggestion
- Update from upstream `coreDNS` v1.6.2 - [changelog](https://coredns.io/2019/09/27/coredns-1.6.4-release/).
```

File: release-notes/aws/v9.0.0.md (hunk 1 above)
Comment (MarcelMue) [reply release-notes/aws/v9.0.0.md]: There could be multiple links e.g. `[0.5.0]` for multiple projects here - does this still work then?

File: release-notes/azure/v8.4.0.md (hunk 2 above)
Comment (MarcelMue) [reply release-notes/azure/v8.4.0.md]: Currently we use past tense everywhere here and we don't use dots at the end unless it is a proper sentence.

I am up for discussion for changing this in general but then I would provide a separate PR and get it straight everywhere.

Comment (kopiczko) [reply release-notes/azure/v8.4.0.md]: How about this https://github.com/giantswarm/releases/pull/32/files#diff-53a25c9124eb60134f8a75b498d5d8c5R16-R18 ?

I already talked with @marians a while ago (during specing VOO) and we agreed to use imperative form. We didn't talk about terminating dots but I think it's fine. Especially if there is a point containing more than one sentence.

Comment (MarcelMue) [reply release-notes/azure/v8.4.0.md]: Okay, I will deal with this in a follow up. I merge here now and then align this afterwards to be imperative (want to show this at engineering day).

**Pull request merged**